OscQueryExample/OscQueryExample.csproj

```diff
@@ -6,6 +6,8 @@
         <Nullable>enable</Nullable>
         <OutputType>Exe</OutputType>
         <LangVersion>12</LangVersion>
+        <!-- Enabled to validate the use of reflection in the project -->
+        <JsonSerializerIsReflectionEnabledByDefault>false</JsonSerializerIsReflectionEnabledByDefault>
```

Comment:
Added to verify that the example project will still run even with reflection disabled for Json Serializer.